### PR TITLE
Wrap build_bifpn_layer in keras layer

### PR DESCRIPTION
**Pull request by Ely-S**
This shares the intention of #323  and #376  in that it seeks to create a keras implementation. This code should be enough to use a BiFPN in a keras model.

I started implementing `build_bifpn_layer` as pure keras code, but the task started to get too big. It might make sense to refactor it into smaller pieces first. Here is what I have working so far, hoping it's a start.

@fsx950223 @mingxingtan Any suggestions on getting it to the next level?

### CODE REVIEW

**Comment by mingxingtan** [comment]
Thank you, @Ely-S and @fsx950223. I agree that we should check in small PRs step by step.

We have two potential ways: (1) create a branch with name "keras" and all changes check to this branch; (2) create a folder name "keras" in master branch, then all these CLs check in to that folder. I personally prefer approach 2 for its simplicity. WDYT?

**Comment by mingxingtan** [comment]
I created a subfolder "keras", could you help move your code to that folder? Thanks! @Ely-S

**Comment by Ely-S** [comment]
@mingxingtan I also like the idea of a submodule because it will allow us to import the latest code from the existing implementation. I'll move this into a keras folder.

**Comment by Ely-S** [comment]
done

**Comment by mingxingtan** [comment]
Thanks for the PR, @Ely-S

File: efficientdet/efficientdet_arch.py [hunk 1]
```diff
@@ -591,6 +591,43 @@ def get_fpn_config(fpn_name, min_level, max_level, weight_method):
...
+    nodes: a list of tensorflow features at different levels
+    weight_method: feature fusion method. One of:
+      * "attn" - Softmax weighted fusion
+      * "fastattn" - Fast normalzied feature fusion
+      * "sum" - a sum of inputs
+  """
+  dtype = nodes[0].dtype
+
+  if weight_method == 'attn':
+    edge_weights = [tf.cast(tf.Variable(1.0, name='WSM'), dtype=dtype)
```
Comment (fsx950223) [inline efficientdet/efficientdet_arch.py:608]: Variable shouldn't be created in ```call```.

**Pull request merged**